ci/cross-pre-build.sh

```
@@ -5,10 +5,13 @@ apt-get update
 apt-get --assume-yes install patch
 
 if [[ $CROSS_TARGET == "x86_64-unknown-linux-gnu" ]]; then
-  apt-get update && apt-get --assume-yes install libssl-dev
+  apt-get update
+  apt-get --assume-yes remove gcc-9
+  apt-get --assume-yes install libssl-dev clang
 fi
 
 if [[ "$CROSS_TARGET" == "aarch64-unknown-linux-gnu" ]]; then
   dpkg --add-architecture $CROSS_DEB_ARCH
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)